react/features/chat/middleware.js

```diff
@@ -152,10 +151,9 @@ StateListenerRegistry.register(
  * @returns {void}
  */
 function _addChatMsgListener(conference, store) {
-    if ((typeof APP !== 'undefined' && !isButtonEnabled('chat'))
```

Comment:
I remember in the past we had a problem with that and that's why it was here.

So in case, the button is hidden but still, someone sends a message. Have you tried this case? 

For example, hide the button from the web UI and from a mobile client where interface config is not used and chat is available, send a message in the chat, does everything work as expected or there are errors in the console?

Reply:
yes I've tested the scenario by joining a meeting with chat button disabled but with iframe API configured and joining the same meeting normally with chat button enabled. Sent both private and public messages from the instance with chat enabled to the instance with chat disabled and did not notice any errors in console. Moreover, once I toggled the chat window through the iframe API on the instance with chat disabled I was able to see the received messages and send private and public messages to the participant from the regular meeting instance.